tests/integration/_experimental/object_detection/test_evaluator_multiclass_f1_optimal.py

```diff
@@ -487,47 +467,47 @@
         y_label="F1-Score",
         curves=[
             Curve(
-                x=[0.1, 0.4, 0.6, 0.7, 0.8, 0.9, 1],
-                y=[15 / 19, 28 / 37, 13 / 18, 24 / 35, 12 / 17, 0.6, 0.4],
+                x=[0.1, 0.4, 0.6, 0.8, 0.9, 1.0],
+                y=[15 / 19, 28 / 37, 13 / 18, 12 / 17, 0.6, 0.4],
                 label="a",
+                extra={
+                    "Precision": [0.75, 14 / 19, 13 / 18, 0.75, 0.75, 5 / 7],
+                    "Recall": [5 / 6, 7 / 9, 13 / 18, 6 / 9, 0.5, 5 / 18],
+                },
             ),
             Curve(
-                x=[0.1, 0.3, 0.4, 0.5, 0.6, 0.7, 0.9, 1],
-                y=[5 / 13, 5 / 12, 10 / 23, 4 / 11, 0.3, 6 / 19, 1 / 3, 0],
+                x=[0.4, 0.5, 0.9, 1],
+                y=[10 / 23, 4 / 11, 1 / 3, 0],
                 label="b",
+                extra={
+                    "Precision": [5 / 11, 0.4, 0.5, 0],
+                    "Recall": [5 / 12, 1 / 3, 1 / 4, 0],
+                },
             ),
             Curve(
                 x=[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.8, 0.9],
                 y=[20 / 23, 16 / 21, 0.7, 12 / 19, 5 / 9, 8 / 17, 0.375, 1 / 7],
                 label="c",
+                extra={
+                    "Precision": [1, 1, 1, 1, 1, 1, 1, 1],
+                    "Recall": [10 / 13, 8 / 13, 7 / 13, 6 / 13, 5 / 13, 4 / 13, 3 / 13, 1 / 13],
+                },
             ),
-            Curve(x=[0.1, 0.6, 0.7, 0.8], y=[8 / 15, 3 / 7, 4 / 13, 1 / 6], label="e"),
-        ],
-        x_config=None,
-        y_config=None,
-    ),
-    CurvePlot(
```

Comment:
Can you point me to the change that would lead to different plots being created in this PR? Looks to me it used to have 2 CurvePlots but now only have 1.

Reply:
I would prefer still verifying it or update verification logic so that we don't need a placeholder of unused fake data. It is a very confusing way to construct test data and to understand.